src/components/Acordition/index.tsx

```diff
@@ -0,0 +1,33 @@
+import type {ReactNode} from 'react';
+import React from 'react';
+import {View} from 'react-native';
+import type {SharedValue} from 'react-native-reanimated';
+import Animated, {useAnimatedStyle, useDerivedValue, useSharedValue, withTiming} from 'react-native-reanimated';
+
+function AccordionItem({isExpanded, children, duration = 300}: {isExpanded: SharedValue<boolean>; children: ReactNode; duration?: number}) {
+    const height = useSharedValue(0);
+
+    const derivedHeight = useDerivedValue(() =>
+        withTiming(height.get() * Number(isExpanded.get()), {
+            duration,
+        }),
+    );
+    const bodyStyle = useAnimatedStyle(() => ({
+        height: derivedHeight.get(),
+    }));
+
+    return (
+        <Animated.View style={[bodyStyle, {overflow: 'hidden'}]}>
```

Comment:
```suggestion
        <Animated.View style={[bodyStyle, styles.overflowHidden]}>
```

where styles come from:
`const styles = useThemeStyles();`